### PR TITLE
feat(comments): allow altering comments

### PR DESCRIPTION
**Pull request by mirkojotic**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [x] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [x] Have you added new tests to prevent regressions?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [x] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/master/CONTRIBUTING.md)?

### Description of change

Solves #10720 
Adds ability to use `changeColumn` to alter comments. It wasn't implemented initially. Works for all dialects except sqlite.

### CODE REVIEW

File: lib/dialects/postgres/query-generator.js [hunk 1]
```diff
@@ -575,7 +576,7 @@ class PostgresQueryGenerator extends AbstractQueryGenerator {
 
     for (const key in attributes) {
       const attribute = attributes[key];
-      result[attribute.field || key] = this.attributeToSQL(attribute, options);
+      result[attribute.field || key] = this.attributeToSQL(attribute, Object.assign({ key }, options || {}));
```
Comment (sushantdhiman) [inline lib/dialects/postgres/query-generator.js]: Why this change?

Comment (mirkojotic) [reply lib/dialects/postgres/query-generator.js]: To create a comment SQL statement in pg, it was necessary to have table name and column the comment is referring to. That line just adds the column name to options.

File: test/integration/query-interface/changeColumn.test.js [hunk 2]
```diff
@@ -219,7 +219,23 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
             expect(describedTable.level_id.allowNull).to.be.equal(true);
           });
         });
-        
+
+        it('should change the comment of column', function() {
+          return this.queryInterface.describeTable({
+            tableName: 'users'
+          }).then( describedTable => {
```
Comment (SimonSchick) [inline test/integration/query-interface/changeColumn.test.js]: Please fix the formatting here.

File: test/integration/query-interface/changeColumn.test.js [hunk 3]
```diff
@@ -219,7 +219,23 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
...
+            expect(describedTable.level_id.comment).to.be.equal(null);
+            return this.queryInterface.changeColumn('users', 'level_id', {
+              type: DataTypes.INTEGER,
+              comment: 'FooBar'
+            });
+          }).then( () => {
+            return this.queryInterface.describeTable({ tableName: 'users' })
+              .then( describedTable2 => {
```
Comment (SimonSchick) [inline test/integration/query-interface/changeColumn.test.js]: Please avoid deep nesting here.

File: test/integration/query-interface/changeColumn.test.js (hunk 2 above)
Comment (sushantdhiman) [reply test/integration/query-interface/changeColumn.test.js]: There is still a space before arguments

Comment (mirkojotic) [reply test/integration/query-interface/changeColumn.test.js]: Should be good now.
